data/virt_autotest/host_unattended_installation_files/autoyast_opensuse_xen_two_disk.xml

```diff
@@ -0,0 +1,147 @@
+<?xml version="1.0"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+  <general>
+    <mode>
+      <confirm t="boolean">false</confirm>
+      <final_reboot t="boolean">true</final_reboot>
+      <second_stage t="boolean">false</second_stage>
+    </mode>
+    <signature-handling>
+      <accept_file_without_checksum t="boolean">true</accept_file_without_checksum>
+      <accept_non_trusted_gpg_key t="boolean">true</accept_non_trusted_gpg_key>
+      <accept_unknown_gpg_key t="boolean">true</accept_unknown_gpg_key>
+      <accept_unsigned_file t="boolean">true</accept_unsigned_file>
+      <accept_verification_failed t="boolean">true</accept_verification_failed>
+      <import_gpg_key t="boolean">true</import_gpg_key>
+    </signature-handling>
+  </general>
+  <report>
+    <errors>
+      <show t="boolean">true</show>
+      <timeout t="integer">20</timeout>
+      <log t="boolean">true</log>
+    </errors>
+    <messages>
+      <show t="boolean">true</show>
+      <timeout t="integer">5</timeout>
+      <log t="boolean">true</log>
+    </messages>
+    <warnings>
+      <show t="boolean">true</show>
+      <timeout t="integer">10</timeout>
+      <log t="boolean">true</log>
+    </warnings>
+    <yesno_messages>
+      <show t="boolean">true</show>
+      <timeout t="integer">30</timeout>
+      <log t="boolean">true</log>
+    </yesno_messages>
+  </report>
+  <bootloader>
+    <loader_type>grub2</loader_type>
+    <global>
+      <activate>true</activate>
+      <boot_mbr>true</boot_mbr>
+      <gfxmode>auto</gfxmode>
+      <os_prober>false</os_prober>
+      <terminal>console serial</terminal>
+      <serial>serial --unit=1 --speed=115200 --word=8 --parity=no</serial>
+      <timeout t="integer">30</timeout>
+      <update_nvram>true</update_nvram>
+      <xen_kernel_append>console=com2,115200 loglvl=all guest_loglvl=all sync_console</xen_kernel_append>
+      <xen_append>console=hvc0,115200 console=tty loglevel=5</xen_append>
+    </global>
+  </bootloader>
+  <partitioning t="list">
+    <drive>
+      <device>/dev/sda</device>
```

Comment:
For xen two disk autoyast, only sda is enough? What's the difference with xen one disk autoyast?

Reply:
The issue was seen only with the KVM guest installation, the xen autoyast profile has <script> which makes it boot into Xen hyperv by default `GRUB_DEFAULT=.*/GRUB_DEFAULT=2/` 

The filename has been changed to meet the naming convention and to avoid confusion, so we run two_disk.xml file for both KVM and Xen on O3 machines or machines with 2 or more disks. And when we want to install on single disk we would use `AUTOYAST=xxx_one_disk.xml` for both Xen and KVM.

When we have machine with 3 or 4 disks and if the issue arises for Xen, we can work it under <partitioning> accordingly in the future, as of now not required. 

This approach allows you to accommodate both scenarios without affecting existing tests. It ensures that tests for machines with different disk configurations can proceed without issues. However, it's important to document and communicate this change to all relevant team members and testers to ensure everyone uses the correct AutoYast profile for their specific testing needs.

Reply:
They basically do the same work as of now.

Reply:
If the issue does not happen on xen, it seems we only need to differentiate the autoyast fo kvm on one or two disk, while for xen better keep it unified as before. HDYT?

Besides, for current kvm issue solution, you are right, at the moment, it is not required for 3 or 4 disks. The point is that this solution won't work seamlessly for any machine. It's known to not work for specific conditions that may not be rare. So, it can only be short term solution. It can be accepted this time since we won't have different machines soon. Please allow me to remind that , in our team, we'd always recommend to think about long term solution as first choice. Look forward to that in future PRs!

Reply:
Is it OK for me either way for Xen like this or like before, @Julie-CAO HDYT?

Reply:
I don't think the two-disk issue happend on kvm host only. It was not revealed on xen most likely it does exist but we did not notice it, how do you know it did not login to a previous snapshot?
So I recommended that xen has 2-disk and 1-disk profiles files both.

Reply:
I don't think the two-disk issue happend on kvm host only. It was not revealed on xen most likely it does exist but we did not notice it, how do you know it did not login to a previous snapshot?
So I recommended that xen has 2-disk and 1-disk profiles files both.

Reply:
okay, sounds reasonable. This does not seem to be related with kvm or xen either.